Gruntfile.js

```diff
@@ -174,6 +174,15 @@ module.exports = function (grunt) {
             },
             'coveralls': {
                 command: 'npm run coveralls'
+            },
+            'coveralls-azure': {
```

Comment:
I don't feel super strongly about this, but to minimize the changes overall, could you leave this file as-is and just call `grunt` (without specifying a task) from azure-pipelines.yml, and then have a conditional step in your azure-pipelines.yml to run coverage or not based on the variable value?

(basically moving the IF logic out of grunt, etc and into the azure-pipelines YAML)

Reply:
I like less files being changed - made this update